components/scripts/lib/gradle.sh

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env bash
 
 invoke_gradle() {
-  local run_num args
-  args=()
+  local run_num
```

Comment:
I suggest declaring envs as a local variable as well.

Reply:
Good idea. Done in 95707cff830c11bb607d5a18abc844c87349330d.